exampleSite/data/features/consulting.yaml

```diff
@@ -1,4 +1,5 @@
 weight: 4
 name: "Consulting"
 icon: "fas fa-lightbulb"
+url: ""
```

Comment:
@salim-b please can you add this parameter to the readme?  here https://github.com/salim-b/hugo-universal-theme/blob/e41d8712415a97e89bd667bf3afce75cf79c0a24/README.md#L283

after that I will merge the PR.

Thanks!

Reply:
Ok, done in https://github.com/devcows/hugo-universal-theme/pull/184/commits/d6c8db5c06dbb450a328b77b2d198800ea349f67